docs/README.md

```diff
@@ -12,6 +12,12 @@ yarn install
 yarn start
 ```
 
+#### Making Changes
+Changes should be made in the `./docs` directory and not in the generated `versioned_docs` or `versioned_sidebars` directories directly.
+
+After the ```yarn start``` command ran successfully you should be able to see your changes in the local [Canary Version](http://localhost:3000/docs/canary/index)
+
+
```

Comment:
Thanks for this! I definitely spent at least 10 minutes refreshing `localhost:3000 trying` to understand why the change wasn't showing up. 🤦 The suggestions below are mostly just stylistic:

```suggestion
#### Making Changes

Changes should be made in the `./docs` directory and not in the generated `./versioned_docs` or `./versioned_sidebars` directories directly.

After running `yarn start`, you should be able to see your changes in the local [Canary Version](http://localhost:3000/docs/canary/index).

```

Reply:
Me too :D That's why I added it hahaha :D